src/io_utils.py

```diff
@@ -0,0 +1,35 @@
+"""io_utils.py
```

Comment:
Consider removing the name of this file within the code comment to help make the code less specific to the module name filename.